### DISCUSSION TITLE
I exceed 100% Vercel parameters

### DISCUSSION THREAD

**Discussion by JIM90mdp** [vercel/community#4762] [Feature Question] [answered]
### Question

Hello, how are you all doing?

Could someone tell me what happens if I exceed 100% of these Vercel parameters?

I've been advised to create a team on Vercel for my client (the owner of the website that generates so much web traffic). I have some doubts because the Vercel documentation suggests that I need to pay for Vercel Pro to create a new team. This leads me to believe that creating a team is not free. Additionally, I'm not sure if I should create the new team from my GitHub account or create a new GitHub account for my client and then create a new team for deploying the website (creating a new GitHub account involves using a new email and phone contact, which I would need to request from the client).

For now, the levels that concern me are those that appear in the image

I apologize for any inconvenience, and I don't want to incur expenses for the client that I didn't consider before selling them the website. I doubt I can fulfill that promise.

Thanks !!!
![image](https://github.com/vercel/community/assets/108559438/a4d7a9e7-6dad-4926-a3cc-894ef579e697)

**Comment by amyegan** [comment, marked as answer]
What happens when you exceed a limit depends on which limit you exceed. Usage areas with a hard limit will result in a temporary pause until the next billing cycle. Some, like image optimization, will result in the feature being disabled without taking down the whole site. Others could result in the site no longer working correctly until the next billing cycle. On a [Pro](https://vercel.com/pricing) team, some features allow for additional usage with on-demand rates. 

You can find more information about Hobby and Pro plan usage limits on our [Limits](https://vercel.com/docs/limits/overview) page and individual feature [documentation](https://vercel.com/docs). 

I also recommend reviewing Vercel's [fair use guidelines](https://vercel.com/docs/limits/fair-use-guidelines) and [Terms of Service](https://vercel.com/legal/terms) since it sounds like you're working with a paying client.

**Reply by rajput-hemant** [reply]
Dear @amyegan,

I hope this message finds you well. I recently initiated a discussion on Vercel's GitHub (https://github.com/orgs/vercel/discussions/4754) regarding a similar issue.

I have over 100 projects deployed under Vercel's hobby plan, and until recently, they consumed around 5/100 GB-Hours for function execution over a year. However, two days ago, in just a few hours, two of my projects caused an inexplicable spike, surpassing 300 GB-Hours and leading to my hobby plan suspension.

I'm struggling to find the reason for this spike, as my site hasn't received such an amount of requests, nor has it taken this much time on function execution. My projects primarily consist of tutorials, university assignments, and personal hobby projects for learning purposes.

I'm reaching out in the hope of receiving your support and insights to resolve this issue and regain access to my hobby plan. As all of my projects have been inaccessible for two days, any guidance to prevent similar overages in the future would be highly appreciated.

Thanks!

**Reply by amyegan** [reply]
@rajput-hemant I just replied to your original post

**Comment by JIM90mdp** [comment]
Hello everyone!

I wanted to share that I've now delegated the deployment responsibilities to the client. Up until now, I had been handling deploys through my Vercel account.

I successfully transferred the domain from my Vercel account to the client's account.

Additionally, I implemented ducanh2912/next-pwa to optimize user requests when accessing the website. This will significantly reduce the bandwidth requirements on Vercel. With this setup, if there are no changes in the cache, the website won't resend requests on subsequent visits, contributing to a more efficient use of resources.

Thank you all so much for your help! I close the thread
